recipes/pandas/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version="0.18.1" %}
+
+package:
+    name: pandas
+    version: {{ version }}
+
+source:
+    fn: pandas-{{ version }}.tar.gz
+    url: https://github.com/pydata/pandas/archive/v{{ version }}.tar.gz
+    sha256: 1f3136f69a58d490ba9f9874e67601042a11138270515dedcf6979c003bba79b
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --root=/
```

Comment:
Sure, but this is also a matter of who is making the fixes.  Computers can
and should fix things like "no trailing newline at EOF" or selectors
missing #.  The setuptools mess can also probably be automated.  Everyone,
including @jakirkham has better things to do than point out or fix those
things.  These are the things that drive me crazy.  There are plenty of
other valuable catches that @jakirkham makes, with his greater knowledge of
effects down the line, such as the OS X SDK target version.  These are the
things that we should be manually inspecting and fixing, not the nits.

On Tue, May 3, 2016 at 2:03 PM, Filipe <notifications@github.com> wrote:

> In recipes/pandas/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/522#discussion_r61937075>
> :
>
> > @@ -0,0 +1,44 @@
> > +{% set version="0.18.1" %}
> > +
> > +package:
> > +    name: pandas
> > +    version: {{ version }}
> > +
> > +source:
> > +    fn: pandas-{{ version }}.tar.gz
> > +    url: https://github.com/pydata/pandas/archive/v{{ version }}.tar.gz
> > +    sha256: 1f3136f69a58d490ba9f9874e67601042a11138270515dedcf6979c003bba79b
> > +
> > +build:
> > +    number: 0
> > +    script: python setup.py install --single-version-externally-managed --root=/
>
> These reviews are death by 1000 cuts. It would be much nicer to put up
> PRs, or add something to the linter so that it provides a link to a "fixed"
> file. Making people go back and forth, and keep track of each little fix
> (which are often pedantic and of questionable functional value) is asking a
> lot people who have a lot to do.
>
> The linter can (and should) be improved, but there are details (and
> discussions) that only a careful human review will catch. I suggest a
> common sense of rule of "good enough to merge," and then a PR to the
> feedstock to fix the remaining details.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/522/files/5fabd3a20813bea17cdffa36fec33106af2db8cc#r61937075>
>